npm/fake_nsp_package.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
```

Comment:
Okay, this can be removed in a separate series of pull requests across repositories.